src/app/core/user/user.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from "@angular/core";
+import { AuthUser } from "../session/auth/auth-user";
+
+@Injectable({
+  providedIn: "root",
+})
+export class UserService {
+  user: AuthUser;
+
+  getCurrentUser(): AuthUser {
+    return this.user;
+  }
+}
```

Comment:
any reason for this to be a service method while loginState is a directly exposed subject?

Reply:
Should become subject now with multi user offline login